sjb/config/test_cases/push_jenkins_images.yml

```diff
@@ -8,6 +8,8 @@ overrides:
     - obulatov@redhat.com
     - agladkov@redhat.com
     - adkaplan@redhat.com
+    - vbobade@redhat.com
+    - abenaiss@redhat.com
```

Comment:
Please remove the Developer Experience team members (one of whom - agladkov - no longer works on the OpenShift project).

Reply:
I have updated, please check